consts.go

```diff
@@ -0,0 +1,16 @@
+package centraldogma
+
+import (
+	"fmt"
+)
+
+var (
+	// ErrLatestNotSet latest not set
```

Comment:
nit: How about removing the comments? I think that they are redundant.

Reply:
I think we should let it like this to by pass linter checking if need in the future 🤔

Reply:
What's the purpose of these comments? Why is it related with linter? How about removing them if doing so does not fail the build?

Reply:
How about just?
```
// Error codes returned by failures in watch.
var (
    ErrLatestNotSet    = fmt.Errorf("latest is not set yet")
    ErrQueryShouldSet  = fmt.Errorf("query should not be nil")
    ErrWatcherIsClosed = fmt.Errorf("watcher is closed")
    ...
)
```
https://golang.org/doc/effective_go.html#commentary

Reply:
Sound good to me. Let me fix it now.

Reply:
Fixed by removing comment. it's not a big deal also 🤔